zconf-ng.h.in

```diff
@@ -22,6 +22,8 @@
 /* Always define z_const as const */
 #define z_const const
 
+typedef size_t z_size_t;
```

Comment:
Do we need it for zlib-ng native API too?

Reply:
Not after we change all the internal usage to use `size_t` directly.

Reply:
Moved definition back to zbuild.h for zlib-ng native api